.github/workflows/notify-on-review-wanted.yml

```
@@ -0,0 +1,37 @@
+name: Notify on Review Wanted
+on:
+  issues:
+    types: [labeled]
+  pull_request_target:
+    types: [labeled]
+
+permissions:
+  contents: read
+
+jobs:
+  notifyOnReviewWanted:
+    name: Notify on Review Wanted
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Determine PR or Issue
+        id: check_pr_or_issue
+        run: |
+          if [[ -n "${{ github.event.pull_request.number }}" ]]; then
+            echo "::set-output name=type::pull_request"
+            echo "::set-output name=number::${{ github.event.pull_request.number }}"
+          else
+            echo "::set-output name=type::issue"
+            echo "::set-output name=number::${{ github.event.issue.number }}"
+          fi
+
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@4e5fb42d249be6a45a298f3c9543b111b02f7907  # 2.3.0
+        env:
+          SLACK_COLOR: '#DE512A'
+          SLACK_ICON: https://github.com/nodejs.png?size=48
+          SLACK_TITLE: ${{ github.actor }} asks for review on ${{ steps.check_pr_or_issue.outputs.type == 'pull_request' && 'PR' || 'Issue' }} #${{ steps.check_pr_or_issue.outputs.number }}
```

```suggestion
          SLACK_TITLE: "${{ github.actor }} asks for review on ${{ steps.check_pr_or_issue.outputs.type == 'pull_request' && 'pull' || 'issue' }} #${{ steps.check_pr_or_issue.outputs.number }}"
```

Otherwise this'll be treated as a comment + rewording